src/main/java/com/minecolonies/coremod/entity/ai/mobs/util/BarbarianSpawnUtils.java

```diff
@@ -163,14 +163,12 @@ else if (barbarian instanceof EntityChiefBarbarian)
      * @param spawnLocation  the location at which to spawn the entity
      * @param world          the world in which the colony and entity are
      */
-    public static void spawn(final String entityToSpawn, final int numberOfSpawns, final BlockPos spawnLocation, final World world)
+    public static void spawn(final String entityToSpawn, final int numberOfSpawns, final BlockPos spawnLocation, final World world, final ForgeChunkManager.Ticket chunkTicket)
     {
-
-        if (spawnLocation != null && entityToSpawn != null && world != null)
+        if (spawnLocation != null && entityToSpawn != null && world != null && !world.isRemote)
         {
             if (!world.isBlockLoaded(spawnLocation))
             {
-                final ForgeChunkManager.Ticket chunkTicket = ForgeChunkManager.requestTicket(MineColonies.instance, world, ForgeChunkManager.Type.NORMAL);
                 if (chunkTicket != null)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Merge this if statement with the enclosing one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1066)